src/index.js

```diff
@@ -46,35 +51,26 @@ const onLoaded = () => {
   handleRouting();
 };
 
-const renderLevelComplete = root => {
-  const completeHTML = `
-  <div id="complete">
-    <h1>Level Complete!</h1>
-      <div id="controls">
-        <button title="Home" id="change-level"><span class="material-symbols-outlined">home</span></button>
-    </div>
-  `;
-  root.innerHTML = completeHTML;
-  document
-    .querySelector("#change-level")
-    .addEventListener("click", () => {
-      location.hash = "";
-     renderMenu(root);
-    })
-  ;
-};
-
 const renderLevel = (root, levelNumber) => {
   const gameHTML = `
+  <div id="overlay-screen"></div>
   <div id="game" class="hide">
     <div id="controls">
       <button title="Undo (z key)" id="undo"><span class="material-symbols-outlined">undo</span></button>
-      <button title="Redo (r key)" id="redo" disabled><span class="material-symbols-outlined">redo</span></button>
+      <button title="Reset (r key)" id="reset"><span class="material-symbols-outlined">refresh</span></button>
       <button title="Home" id="change-level"><span class="material-symbols-outlined">home</span></button>
-      <button title="Settings" disabled><span class="material-symbols-outlined">settings</span></button>
-      <button title="Help" disabled><span class="material-symbols-outlined">help</span></button>
+      <button title="Help" id="help"><span class="material-symbols-outlined">help</span></button>
     </div>
     <div id="board"></div>
+    <div id="status">
+      <div>Level <span id="level-label"></span></div>
+      <button title="Stats" id="collapse"><span class="material-symbols-outlined">keyboard_double_arrow_down</span></button>
+      <div id="stats">
+        <div>current:  <span id="current"></span></div>
```

Comment:
```suggestion
        <div>current: <span id="current"></span></div>
```
Multiple spaces are rendered as 1 in HTML